turbopack/crates/turbopack-ecmascript/src/references/external_module.rs

```diff
@@ -180,6 +186,12 @@ pub struct CachedExternalModuleChunkItem {
     chunking_context: ResolvedVc<Box<dyn ChunkingContext>>,
 }
 
+// Without this wrapper, VirtualFileSystem::new_with_name always returns a new filesystem
+#[turbo_tasks::function]
+fn external_fs() -> Vc<VirtualFileSystem> {
+    VirtualFileSystem::new_with_name("externals".into())
+}
```

Comment:
I dislike using virtual file system for traced and externals. I hope that's fixed later in the stack...

Reply:
Yes, in https://github.com/vercel/next.js/pull/72517 👍